integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -779,30 +836,15 @@ WEBLOGIC_IMAGE_TO_USE_IN_SPEC, adminSecretName, createSecretsForImageRepos(domai
       checkServiceDoesNotExist(managedServerPrefix + i, domainNamespace);
     }
 
-    // delete domain3
+    // delete domain9
     deleteDomainResource(domainNamespace, domainUid);
-    String configMapName1 = "modelfiles1-cm";
-
-    //add model files to configmap and verify domain is running
-    cmFiles.add(
-            Paths.get(MODEL_DIR + "/" + MII_BASIC_WDT_MODEL_FILE));
-    cmFiles.add(Paths.get(MODEL_DIR + "/multi-model-one-ds.20.yaml"));
-    assertDoesNotThrow(
-            () -> createConfigMapForDomainCreation(
-                    configMapName1, cmFiles, domainNamespace, this.getClass().getSimpleName()),
-            "Create configmap for domain creation failed");
-
-    // create domain custom resource using auxiliary image
-    logger.info("Creating domain custom resource with domainUid {0} and auxiliary image {1}",
-            domainUid, miiAuxiliaryImage12);
-    DomainResource domainCR1 = createDomainResourceWithAuxiliaryImage(domainUid, domainNamespace,
-            WEBLOGIC_IMAGE_TO_USE_IN_SPEC, adminSecretName, createSecretsForImageRepos(domainNamespace),
-            encryptionSecretName, replicaCount, auxiliaryImagePathCustom,
-            miiAuxiliaryImage12);
-    assertNotNull(domainCR1, "failed to create domain resource");
-    domainCR1.spec().configuration().model().configMap(configMapName1);
-    createDomainAndVerify(domainUid, domainCR1, domainNamespace,
-            adminServerPodName, managedServerPrefix, replicaCount);
+    deleteConfigMap(configMapName, domainNamespace);
+    testUntil(
+        withLongRetryPolicy,
+        () -> listConfigMaps(domainNamespace).getItems().stream().noneMatch((cm)
+            -> (cm.getMetadata().getName().equals(configMapName))),
+        logger,
+        "configmap {0} to be deleted.");
```

Comment:
You need to add configmap here.
testUntil(
         withLongRetryPolicy,
         () -> listConfigMaps(domainNamespace).getItems().stream().noneMatch((cm)
             -> (cm.getMetadata().getName().equals(configMapName))),
         logger,
         "configmap {0} to be deleted",
         configMapName);